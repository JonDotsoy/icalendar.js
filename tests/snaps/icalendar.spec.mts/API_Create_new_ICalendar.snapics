ICalendar {
  kind: 'VCALENDAR',
  properties: Map(1) {
    'FOO' => PropertyValue { value: { '$Text': 'asd' }, parameters: Map(0) {} }
  },
  components: Set(1) {
    VComponent {
      kind: 'EVENT',
      properties: Map(0) {},
      components: Set(0) {}
    }
  }
}