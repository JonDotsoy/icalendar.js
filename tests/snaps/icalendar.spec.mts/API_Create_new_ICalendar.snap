ICalendar {
  kind: 'VCALENDAR',
  properties: Map(0) {},
  components: Set(0) {}
}